result/yield-star-spacing.js

```diff
@@ -0,0 +1,24 @@
+/*eslint yield-star-spacing: ["error", "after"]*/
+/*eslint-env es6*/
+
+function* generator() {
```

Comment:
__[development][eslint]__
* Error - 'generator' is defined but never used (no-unused-vars)